tests/integration/workflows/conftest.py

```diff
@@ -224,8 +227,21 @@ def mocked_external_services(workflow_app):
             status_code=200,
             text='Irrelevant part 1 of message \nIrrelevant part 2 of message \n# Ticket 1 updated.'
         )
-
-        yield
+        if 'INSPIREHEP_URL' in workflow_app.config:
```

Comment:
why not `vcr`? :D

Reply:
This is much simpler than creating a record to external service (which on my local setup I cannot even run together with next). Additionally it's not necessary to have proper response from HEP here, as we only want to check if we send correct stuff.